README.md

```diff
@@ -4,74 +4,97 @@
 
 ## Sobre
 
-Alectrion é um sistema de controle e gerenciamento de inventário e ordens de serviços de TI.
+Alectrion é um sistema de gerenciamento de inventário, ordens de serviços e movimentações de equipamentos de informática,
+feito para atender as demandas da DSTI (Divisão de Suporte Técnico Interno) da Polícia Civil do estado de Goiás.
 
-Aplicação disponível em: [link da aplicação](https://alectrion-front.herokuapp.com/)
+Aplicação disponível em: TO-DO
 
 ## Requisitos
-- Local(sem docker)
+
+- Local (sem docker)
     - node (versão 16.10.0 ou superiores)
     - yarn (versão 1.22.18)
+
 - Docker
     - Docker e docker-compose
 ## Instalação e Execução
 
 1. Clone o projeto 
 
-    > git clone https://github.com/fga-eps-mds/2022-1-Alectrion-Gateway
+> git clone https://github.com/fga-eps-mds/2022-2-Alectrion-Gateway
 
 2. Entre na pasta do projeto
 
-    > cd 2022-1-Alectrion-Gateway
+> cd 2022-2-Alectrion-Gateway
 
 3. Crie um arquivo .env(variaveis de exemplo no .env.example deste repositório) adicionando as URL para os serviços
  UserApi e EquipamentApi.
 
-### Local(Sem docker)
+### Local (Sem docker)
 
-4. Instale as depencencias
+4. Instale as dependências
         
-    > yarn
+> yarn
 
 6. Crie um arquivo .env da mesma forma do arquivo .env.example
 
 7. Execute o projeto
 
-    > yarn dev
+> yarn dev
 
 
 ### Docker
 
 4. Gerar imagem
 
-    > docker build -t alectrion-gateway .
+> docker build -t alectrion-gateway .
 
 6. Rodar container
     
-    > docker run --env-file .env -p 4000:4000 -d --name alectrion-gateway-container alectrion-gateway
+> docker run --env-file .env -p 4000:4000 -d --name alectrion-gateway-container alectrion-gateway
 
```

Comment:
Atualmente o docker não está funcionando como deveria, melhor apagar essa parte e colocar de volta quando esse problema for resolvido.

Reply:
Done